pkg/controllers/dynakube/activegate/reconciler.go

```diff
@@ -107,12 +107,10 @@ func (r *Reconciler) Reconcile(ctx context.Context) error {
 		return err
 	}
 
-	if r.dynakube.NeedsActiveGate() {
-		if r.istioReconciler != nil {
-			err = r.istioReconciler.ReconcileActiveGateCommunicationHosts(ctx, r.dynakube)
-			if err != nil {
-				return err
-			}
+	if r.istioReconciler != nil {
+		err = r.istioReconciler.ReconcileActiveGateCommunicationHosts(ctx, r.dynakube)
+		if err != nil {
```

Comment:
useless error checking, just return `r.istioReconciler.ReconcileActiveGateCommunicationHosts(ctx, r.dynakube)`

Reply:
fixed in c4c2371a

Reply:
Yep, Werner is right. The function continues after this.